CHANGELOG.md

```diff
@@ -2,18 +2,36 @@
 
 ## Unreleased
 
+## v0.6.0 Beta
+
 ## 🛑 Breaking changes 🛑
 
 - Renamed the metrics generated by `hostmetrics` receiver to match the (currently still pending) OpenTelemetry system metric conventions (#1261) (#1269)
-
-## 🚀 New components 🚀
+- Removed `vmmetrics` receiver (#1282)
+- Removed `cpu` scraper `report_per_cpu` config option (#1326)
 
 ## 💡 Enhancements 💡
 
 - Added disk merged (#1267) and process count (#1268) metrics to `hostmetrics`
+- Log metric data points in `logging` exporter (#1258)
+- Changed the `batch` processor to not ignore the errors returned by the exporters (#1259)
+- Build and publish MSI (#1153) and DEB/RPM packages (#1278, #1335)
+- Added batch size metric to `batch` processor (#1241)
+- Added log support for `memorylimiter` processor (#1291) and `logging` exporter (#1298)
+- Always add tags for `observability`, other metrics may use them (#1312)
+- Added metrics support (#1313) and allow partial retries in `queued_retry` processor (#1297)
+- Migrate `resource` processor to the new OTLP-based internal data model (#1315)
```

Comment:
Could you rephrase it to highlight interface changes. (I should've put that in commit header)

```suggestion
- Update resource processor: introduce `attributes` config parameter to specify actions on attributes similar to attributes processor, old config interface is deprecated (#1315)
```